### PR TITLE
erlang/accumulate: add mentoring notes

### PR DESCRIPTION
**Pull request by NobbZ**

### CODE REVIEW

File: tracks/erlang/exercises/accumulate/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#### Fold
+
+```erl
+accumulate(F, L) -> lists:foldr(fun(E, Acc) -> [F(E)|Acc] end, [], L).
+```
+
+### Common suggestions
+
+* If the student uses a comprehension, ask him to use recursion, if he
+  uses recursion ask him to use a fold.  But do not push to hard.  The
```
Comment (kotp) [inline tracks/erlang/exercises/accumulate/README.md]:
```diff
1c1
< do not push to hard
---
> do not push too hard
```

File: tracks/erlang/exercises/accumulate/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  main subject of this exercise is to show the student, that functions
```
Comment (kotp) [inline tracks/erlang/exercises/accumulate/README.md]: the comma should not be there.

File: tracks/erlang/exercises/accumulate/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  are just values and you can pass them around and call them.
+* In recursive versions, it is important to check if the
+  implementation is indeed tail recursive.  When building a list it is
+  not that important (we move allocations from stack to heap and still
+  need to reverse afterwards), but tailrecursiveness is a good habbit
```
Comment (kotp) [inline tracks/erlang/exercises/accumulate/README.md]:
```diff
1c1
<   need to reverse afterwards), but tailrecursiveness is a good habbit
---
>   need to reverse afterwards), but tail recursiveness is a good habit
```

**Comment by NobbZ** [comment]
Thank you @kotp, I fixed them.

**Comment by ErikSchierboom** [comment]
@kotp could you look at this PR to see if it is in a state where it can be merged?

**Comment by ErikSchierboom** [comment]
Merged, thanks!

**Pull request merged**